ta/pkcs11/src/processing.c

```diff
@@ -1268,6 +1268,7 @@ enum pkcs11_rc entry_wrap_key(struct pkcs11_client *client,
 out:
 	release_active_processing(session);
 out_free:
+	TEE_Free(key_data);
```

Comment:
it is quite tricky to understand that this free is related to  `get_key_data_to_wrap()` behaviour.
I suggest to rename `get_key_data_to_wrap()` to something like `alloc_key_data_to_wrap()` and an explicit inlien comment description, for example in _ta/pkcs11/src/pkcs11_attributes.c_:
```c
/*
 * Get an allocated copy of key data to be wrapped in @head
 * @head: Object attribute where to find key data to be wrapped
 * @data: Output allocated and filled buffer upon success
 * @sz: Key Output data size in byte upon success
 * Return a pkcs11_rv comliant value
 */
enum pkcs11_rc alloc_key_data_to_wrap(struct obj_attrs *head, void **data,
				    uint32_t *sz)
{
    ...
}
```

Reply:
applied,
thanks for providing solution